src/btree.c

```diff
@@ -3083,13 +3083,59 @@ btree_print_offset_table(platform_log_handle *log_handle, btree_hdr *hdr)
    platform_log(log_handle, "\n");
 }
 
+static void
+btree_print_btree_pivot_stats(platform_log_handle *log_handle,
+                              btree_pivot_stats   *pivot_stats)
+{
+   // If nothing has been updated, yet, print a msg accordingly.
+   if ((pivot_stats->num_kvs == BTREE_UNKNOWN_COUNTER)
+       && (pivot_stats->key_bytes == BTREE_UNKNOWN_COUNTER)
+       && (pivot_stats->message_bytes == BTREE_UNKNOWN_COUNTER))
+   {
+      platform_log(log_handle, "   (Pivot Stats=BTREE_UNKNOWN_COUNTER)\n");
```

Comment:
The only real change in output, other than modularizing inline code into fns, is this line.

In a follow-on PR #512 , I am updating lines 3105 and 3106 below to print the # of bytes also with unit-specifier strings.

For the default case when BTree stats are not gathered, all values are 'uninitialized' to this large `BTREE_UNKNOWN_COUNTER` value (i.e., 2 billion something). I did not want to continually misreport these values as '2.xx GiB' as that's misleading.

Hence, this check above to report when all values are still at their default uninitialized state.

Reply:
Would it be better to check each counter separately?

Reply:
It's a little bit messy to do that right now. Also, I think, in places where these stats are accumulated, say in `btree_accumulate_pivot_stats()`, we will update all 3.

From running tests, I found that either all 3 are uninitialized or they will have some valid value.

So did it this way. In an upcoming change, I am going to print the values as a string. At that time, I will revisit if I can check each value and print `"BTREE_UNKNOWN_COUNTER"`, as appropriate.